app/assets/stylesheets/profile-preview-card.scss

```diff
@@ -6,11 +6,13 @@
     left: 0;
   }
 
+  // Dropdown hidden state is controlled by an inline style,
+  // !important is valid in this context to override any 'normal' dropdown behaviors
   &__content.crayons-dropdown:hover {
-    display: block;
+    display: block !important;
```

Comment:
I wouldn't normally advocate for use of `!important` but I think in this context, where we are temporarily overriding core/default functionality for a purely visual effect, it makes sense.

Reply:
Nitpick (non-blocking): `!important` is fine in this case, but if possible, try and set a `hidden` class instead of inlining `display: none`. I'm still trying to see what sets it. 

https://github.com/forem/forem/blob/4da6fc096cbb9c4ba6cc60cf6017942f614cf77c/app/javascript/utilities/dropdownUtils.js#L123

and remove that class when necessary, instead of explicitly inlining `display: block`.

https://github.com/forem/forem/blob/4da6fc096cbb9c4ba6cc60cf6017942f614cf77c/app/javascript/utilities/dropdownUtils.js#L92

Reply:
I'm thinking maybe an alternative could be:

- Inline the `dropdownContent.style.display = 'block'; `
- Remove the inline style when we hide it again (rather than setting `display: none`): `dropdownContent.style.removeProperty('display')`

My reasoning is that relying on a `hidden` class would mean we'd have to update every instance of a crayons dropdown to have a separate `hidden` class, whereas at the moment it's baked into the crayons dropdown styles. I'm also a bit concerned about ending up in a CSS specificity war if we don't use an inline style to display the dropdown 😅 

If we avoid inlining `display: none` then I think we don't need any other `!important` CSS which I agree is a lot neater 👍